src/app/about/officer-data.ts

```diff
@@ -1,394 +1,148 @@
-        icon: "/linkedin_logo.png",
-      },
-    ],
+    socials: {},
   },
-  andrew_kennedy: {
-    position: "",
-    image: "/ssd_logo.svg",
-    name: "Andrew Kennedy",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "https://www.linkedin.com/in/andrewphkennedy/",
-        icon: "/linkedin_logo.png",
-      },
-    ],
+  stephen_coomes: {
+    name: "Stephen Coomes",
+    image: "public/officers/Stephen_Coomes.jpg",
+    socials: {
+      LinkedIn: "https://linkedin.com/in/stephen-coomes-8a4885221",
+    },
   },
   tianrui_li: {
-    position: "",
-    image: "public/officers/tianrui_li.jpg",
     name: "Tianrui Li",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "https://www.linkedin.com/in/tianrui-li-b7343518a/",
-        icon: "/linkedin_logo.png",
-      },
-    ],
-  },
-  katherine_chan: {
-    position: "",
-    image: "/ssd_logo.svg",
-    name: "Katherine Chan",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "",
-        icon: "/linkedin_logo.png",
-      },
-    ],
-  },
-  blake_anderson: {
-    position: "",
-    image: "public/officers/blake_anderson.jpg",
-    name: "Blake Anderson",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "https://www.linkedin.com/in/willbanders/",
-        icon: "/linkedin_logo.png",
-      },
-    ],
+    image: "public/officers/Tianrui_Li.jpg",
+    socials: {
+      LinkedIn: "https://linkedin.com/in/tianrui-li-b7343518a",
+    },
   },
-  song_li: {
-    position: "",
-    image: "/ssd_logo.svg",
-    name: "Song Li",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "",
-        icon: "/linkedin_logo.png",
-      },
-    ],
+  trevor_richardson: {
+    name: "Trevor Richardson",
+    image: "public/officers/Trevor_Richardson.jpg",
+    socials: {
+      LinkedIn: "https://linkedin.com/in/trevorarichardson",
+    },
   },
-  scott_harrison: {
-    position: "",
-    image: "/ssd_logo.svg",
-    name: "Scott Harrison",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "",
-        icon: "/linkedin_logo.png",
-      },
-    ],
+  varun_puri: {
+    name: "Varun Puri",
+    image: "public/officers/Varun_Puri.jpg",
+    socials: {
+      LinkedIn: "https://linkedin.com/in/varun-puri",
+    },
   },
-  jacob_bleser: {
-    position: "",
-    image: "/ssd_logo.svg",
-    name: "Jacob Bleser",
-    socialLinksData: [
-      {
-        name: "LinkedIn",
-        url: "",
-        icon: "/linkedin_logo.png",
-      },
-    ],
+  yonas_bahre: {
+    name: "Yonas Bahre",
+    image: "public/officers/Yonas_Bahre.jpg",
+    socials: {
+      GitHub: "https://github.com/yonasbahre",
+      LinkedIn: "https://linkedin.com/in/yonasbahre",
+    },
   },
 };
 
-export const positionData: {
-  [key: string]: { [key: string]: OfficerCardProps };
-} = {
+export const positionData: Record<string, Record<string, OfficerData>> = {
```

Comment:
[Comment]: This is actually an interesting case - above I changed socials to be an map instead of an array, here I probably would have gone in the other direction. Having positions be a map feels a little weirder to me; in part because there's not really any mapping to look up values here - rather it fully relies on iteration order. I think the fact that the keys have spaces too certainly doesn't help this.

Either way; let's leave this as is - any change is a style preference and doesn't have any additional benefit/savings right now.